debian/libfabric1.install

```diff
@@ -0,0 +1,4 @@
+usr/bin/*
+usr/lib/*/libfabric*.so.*
+usr/share/man/man1
+usr/share/man/man7
```

Comment:
Did this file come from upstream?  Is Ubuntu packaging binaries and manpages in their library package?

I believe this violates the library packaging principle that multiple versions of the same library (i.e. libfabric1, libfabric2, libfabric3, etc.) should be installable at the same time.

Reply:
This came from the Ubuntu libfabric 1.5.3-1 packaging.

Reply:
This is bad form on their part -- violating the packaging standards, and a pity indeed.